spinalcordtoolbox/vertebrae/core.py

```diff
@@ -473,13 +541,16 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, verbose=1):
     """
 
     # open segmentation
+    # sct.run('sct_resample -i %s -mm 1x1x1 -x nn -o segmentation_straight.nii'%(fname_seg))
+    # fname_seg = 'segmentation_straight.nii'
```

Comment:
Nitpick: unused commented commands (cleanup?)

Reply:
Yes definitely.

Reply:
done in ec1c020